concepts/methods/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Methods are similar to functions, but are associated with a particular type and can be called on values of the type they are associated.",
```

Comment:
Simplification:

Methods are functions with a special receiver argument that can be called on an instance of the receiver type.